applications/nrf5340_audio/src/modules/led_assignments.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _LED_ASSIGNMENTS_H_
+#define _LED_ASSIGNMENTS_H_
+
+#include <zephyr/devicetree.h>
+#include <stdint.h>
+
+#define LED_AUDIO_NOT_ASSIGNED 0xFF
+
+/**
+ * @brief Mapping of LEDs on board to indication.
+ */
+#if DT_NODE_EXISTS(DT_ALIAS(led_device_type))
```

Comment:
This is ok, but we can see if we can find a more dynamic way of handling this.

Reply:
Sorry I don't understand this comment.